backend/battling/api/urls.py

```diff
@@ -8,4 +8,5 @@
     path("battles/<int:pk>/", endpoints.BattleDetail.as_view(), name="battle-detail"),
     path("battles/create/", endpoints.CreateBattle.as_view(), name="create-battle"),
     path("team/<int:pk>/edit/", endpoints.CreateTeam.as_view(), name="create-team"),
+    path("user/", endpoints.LoggedUser.as_view(), name="logged-user"),
```

Comment:
Since this is an API endpoint, this should be prefixed with `api/`, also shouldn't this be in another file?

Reply:
It is prefixed with api... Inside my urls in the main url files I have a "path("api/", include("battling.api.urls")),"